docs/sources/administration/team-management/index.md

```diff
@@ -23,7 +23,7 @@ A team is a group of users within an organization that have common dashboard and
 
 A user can belong to multiple teams.
 
-### Before you begin
+**Before you begin:**
```

Comment:
Aren't these actually headings? If so, shouldn't they semantically be marked as such? Why were they changed?